common/mod/ngv/cadifus.mod

```diff
@@ -363,7 +364,18 @@ BREAKPOINT {
     :i=0
     caerAll=caer[0]
 
-    if (glu2>0){
+    is_glu2_assigned = 0
+: here we want to check glu2 assignment. checking _p_glu2 ends in 
+: RuntimeError: hocobj_call error: generic_data_handle{cont=GlutReceive 
+: glut row=0/129 type=double*}::literal_value<void*> cannot be called on 
+: a handle [that was] in modern mode

```

Comment:
what is `modern mode`?

Reply:
Ah: I see; that whole stanza is from the error - it wasn't clear where it ended.

Will this comment make sense, then? Why would someone try access `_p_glu2` if it's no longer supported?

Reply:
it is still there and that way of accessing stuff `_p_*` is still used all around the code. It is to warn future developers that checking `_p_glu2` does not work